### ISSUE TITLE
example tests don't run as part of CI

### ISSUE THREAD

**Issue by chrisns** [UKHomeOfficeForms/hof-bootstrap#96]
In example/ you've got codecept there, but the tests don't run as part of CI

**Comment by JoeChapman** [comment]
@lennym @easternbloc ideas for how to do this nicely?

**Comment by lennym** [comment]
`rm -rf example`?

**Comment by JoeChapman** [comment]
And any new examples? It's not a bad idea to run tests against the examples when HOF core changes

**Comment by lennym** [comment]
https://github.com/UKHomeOfficeForms/hof-examples?

**Comment by JoeChapman** [comment]
Yes those, the examples that don't have tests. Maybe they should have tests. However, if your point is "they don't have tests", then sure forget it. Close this issue

**Comment by lennym** [comment]
I think we should remove the example in this repo in favour of a single-source-of-truth in the other repo.

As for testing those examples... it's not a bad idea. It's just more work to be done. By all means add functional tests for examples when creating them.
